### PR TITLE
#161966897 Test user login process (no social authentication)

### PR DESCRIPTION
**Pull request by codeplus254**
#### What does this PR do?
Tests the normal login process where the user inputs their email and id.
#### Description of Task to be completed?
Write failing tests before the login feature is implemented. 
#### How should this be manually tested?

- After cloning the repo, CD into it. 

- Create and start your virtual environment 

- Install dependencies. `pip3 install -r requirements.txt`

- Define the following environment variables and place them in a `.env` file in the root directory

> 

`export DATABASE_USER='postgres'
export DATABASE_PASSWORD='postgres'
export HOST='localhost'
export PORT='5432'
export DATABASE_TEST='test_db' `

>

- Add the variables into the virtual environment you just created. `source .env`

- run `python3 manage.py test`. If the tests cannot be found automatically, you'll need to add the url after the word test (  `python3 manage.py test authors.apps.authentication.tests.login_tests` )
#### Any background context you want to provide?
The user will have login into the app by providing email and password.
#### What are the relevant pivotal tracker stories?
[#161966897](https://www.pivotaltracker.com/story/show/161966897)
#### Screenshots (if appropriate)
screenshot 2018-12-06 at 11 54 20

### CODE REVIEW

File: authors/apps/authentication/tests/login_tests.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+    def test_inexistent_user(self):
+        """This function tests whether an unregistered user can login"""
+        response = self.client.post(
+            'api/users/login',
+            data=json.dumps(self.test_unknown_user),
+            format='json')
+
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+    def test_wrong_password(self):
```
Comment (r-wambui) [inline authors/apps/authentication/tests/login_tests.py]: Kindly make the function names more descriptive. For example `test_cannot_login_with_invalid_credentials`

Comment (r-wambui) [reply authors/apps/authentication/tests/login_tests.py]: There was a task on PT with JWT, did you cater for this?

Comment (codeplus254) [reply authors/apps/authentication/tests/login_tests.py]: Not yet, I was just doing the login process

Comment (r-wambui) [reply authors/apps/authentication/tests/login_tests.py]: Yes am aware of this, though when login we should have the token generated. My point is remember to cater for token while working on the user module tests

Comment (leonardnjura) [reply authors/apps/authentication/tests/login_tests.py]: Thanks @codeplus254 we will really need this feature

**Pull request closed without merging**